docs/running-on-kubernetes.md

```diff
@@ -83,7 +83,7 @@ the cluster.
 ### Specifying input files
 
 Spark supports specifying JAR paths that are either on the submitting host's disk, or are located on the disk of the
-driver and executors. Refer to the [application submission](submitting-applications.html#advanced-dependency-management)
+driver and executors. Refer to the [application submission](submitting-applications.md#advanced-dependency-management)
```

Comment:
The links should be .html - it's for the generated static site, rather than meant to be visited on github.